### PR TITLE
Adds the Ocelot traitor bundle

### PR DESCRIPTION
**Pull request by datlo**
Remade this PR because commits got messed up, see previous comments here https://github.com/ParadiseSS13/Paradise/pull/24285

-----

## What Does This PR Do

Adds the Ocelot bundle as an option for syndicate bundles. Contains two syndi revolver, two speedloaders, combat knife, encryption key, a holster, and appropriate outfit. Also fixed the TC cost comments on payday bundle.
image

## Why It's Good For The Game

Would be a cool reference and addition to syndi bundles, dual wield is fun. You could also trade with a fellow traitor to arm yourselves both. The Holo-cigar item was intentionally *not* part of this bundle for the sake of 1. balance 2. keeping human specific item to humans

E4IyrCPVUAUxvSr

## Testing

Spawned a buncha bundle beacons till I found the new ocelot bundle, spawned it, checked that everything was there.

## Changelog
:cl:
add: Added the Ocelot bundle to syndicate bundle selection.
/:cl:

### CODE REVIEW

**Comment by Danchi299** [comment]
kit with no utility, lots of damage, and no stealth potential.
because revolvers are cool!

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -91,23 +91,23 @@
 		/obj/item/autosurgeon/organ/syndicate/meson_eyes, // 0TC
 		/obj/item/encryptionkey/syndicate) // 2TC
 
-/// 175TC
+/// 185TC
 /obj/item/storage/box/syndie_kit/bundle/payday
 	name = "Heist Bundle"
 	desc = "Alright guys, today we're performing a heist on a space station owned by a greedy corporation. Drain the vault of all its worth so we can get that pay dirt!11"
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/uplink_kits.dm:97]: "dirt!11" do you think this is a typo on the description?

**Review by Henri215 — APPROVED** [review]
Lewcing good to me

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 1 above)
Comment (datlo) [reply code/game/objects/items/weapons/storage/uplink_kits.dm:97]: I counted the items, adds up to the amount. I guess I could be wrong but I like triple checked.

Comment (Henri215) [reply code/game/objects/items/weapons/storage/uplink_kits.dm:97]: Learning a new meme with each day

**Review by DGamerL — APPROVED** [review]

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 1 above)
Comment (Danchi299) [reply code/game/objects/items/weapons/storage/uplink_kits.dm:97]: "!11" is not a typo, it was copied from the syndicate equipment wiki page, as descriptions come from there first

**Review by S34NW — APPROVED** [review]

**Pull request merged**